kurtosis-devnet/pkg/kurtosis/kurtosis.go

```diff
@@ -162,10 +170,17 @@ func (d *KurtosisDeployer) GetEnvironmentInfo(ctx context.Context, spec *spec.En
 		return nil, fmt.Errorf("failed to extract JWT data: %w", err)
 	}
 
+	// Get dependency set
+	depsetData, err := d.depsetExtractor.ExtractData(ctx, d.enclave)
```

Comment:
thoughts on https://github.com/ethereum-optimism/optimism/issues/15048 to solve this?

Reply:
This is the full copy of the dependency set config, not a snapshot.
We can maybe reduce it to a snapshot-getter in the test frontend, but the go in-process backend counterpart needs the full one, to be able to spin up additional supervisor nodes.

And kurtosis is also being extended to support more supervisor nodes, so I think it's useful to have in the underlying kurtosis package.

And then once it's available, the test frontend can present it to the test itself also, like the rollup config and chain configs are, so the test is aware of what's configured.

Reply:
I think if we can avoid passing that dependency set around, and have it discovered/computed instead, then it's great to have the tests do just that.

But as long as that's effectively a deployment artifact, I'd rather take it from a static location than communicating with the system under test, as a bug in that part of the system might then prevent unrelated tests from running.